evap/evaluation/models.py

```diff
@@ -1452,6 +1471,12 @@ class Meta:
         ordering = ["id"]
         verbose_name = _("text answer")
         verbose_name_plural = _("text answers")
+        constraints = [
+            CheckConstraint(
+                check = ~Q(answer=F('original_answer')),
```

Comment:
This could also be written as
```suggestion
                check = Q(answer__ne=F('original_answer')),
```
How do we feel about both versions? @He3lixxx ?

Reply:
It seems to be impossible on text fields.